Add cstdint as necessary

In GCC 13.1 usage of uint64 and similar will result in an error without
 #include <cstdint>

Signed-off-by: Mark Hatle <mark.hatle@amd.com>

Index: src/runtime_src/core/edge/user/aie/common_layer/adf_api_config.h
===================================================================
--- a/src/runtime_src/core/edge/user/aie/common_layer/adf_api_config.h
+++ b/src/runtime_src/core/edge/user/aie/common_layer/adf_api_config.h
@@ -18,6 +18,7 @@
 
 #include <string>
 #include <vector>
+#include <cstdint>
 
 namespace adf
 {
Index: src/runtime_src/core/edge/user/zynq_dev.h
===================================================================
--- a/src/runtime_src/core/edge/user/zynq_dev.h
+++ b/src/runtime_src/core/edge/user/zynq_dev.h
@@ -19,6 +19,7 @@
 #include <fstream>
 #include <string>
 #include <vector>
+#include <cstdint>
 
 class zynq_device {
 public:
 
